### PR TITLE
LPS-136538

### PR DESCRIPTION
**Pull request by Odrakir1**
The issue was solved calling another normalizer function instead of the one that wouldn't remove the friendly URL's accents.

### CODE REVIEW

**Comment by Odrakir1** [comment]
ci:test:sf

**Comment by Odrakir1** [comment]
ci:test:relevant

**Comment by Odrakir1** [comment]
ci:test:sf

**Comment by wanderlast** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java [hunk 1]
```diff
@@ -82,10 +82,11 @@ public String getFriendlyURL(
 		friendlyURL = getFriendlyURL(friendlyURL);
 
 		if (Validator.isNotNull(friendlyURL)) {
-			return friendlyURL;
+			return FriendlyURLNormalizerUtil.normalize(friendlyURL);
 		}
 
-		friendlyURL = StringPool.SLASH + getFriendlyURL(name);
```
Comment (wanderlast) [inline portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java:88]: getFriendlyURL was removed here, but not earlier. Is that intentional?

File: portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java [hunk 2]
```diff
@@ -82,10 +82,11 @@ public String getFriendlyURL(
 		friendlyURL = getFriendlyURL(friendlyURL);
```
Comment (wanderlast) [inline portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java]: Considering what getFriendlyURL() is, is there a point to normalizing twice?

**Review by wanderlast — COMMENTED** [review]
Hi @Odrakir1,

Could you please take a look at my two comments and let me know what you think? Thanks!

**Comment by Odrakir1** [comment]
Hi @wanderlast,

Thank you for your observations, that just passed unnoticed by me. I'm pushing my changes. Also, sorry for the delay.

File: portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java (hunk 2 above)
Comment (Odrakir1) [reply portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java]: `getFriendlyURL()` is also responsible for editing the friendly URL. So the friendly URL is based on the `name` variable when we are creating a new page, but when we are editing the page's friendly URL we normalize `friendlyURL` which is the new value for friend URL the user set.

Comment (wanderlast) [reply portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java]: I'm assuming this comment was meant for the other conversation, but that's fair.

**Review by wanderlast — COMMENTED** [review]
No worries, take as much time as you need.

Isn't the behavior between getFriendlyURL(long groupId, boolean privateLayout, long layoutId, String name, String friendlyURL) and getFriendlyURL(String friendlyURL) now inconsistent though? Should these be returning two different results?

**Comment by Odrakir1** [comment]
> No worries, take as much time as you need.
> 
> Isn't the behavior between getFriendlyURL(long groupId, boolean privateLayout, long layoutId, String name, String friendlyURL) and getFriendlyURL(String friendlyURL) now inconsistent though? Should these be returning two different results?

The `getFriendlyURL(String friendlyURL)` was being used by `getFriendlyURL(long groupId, boolean privateLayout, long layoutId, String name, String friendlyURL)`, but as you recommended since it is a public API, I'm changing it to keep consistency. 

Thank you!

**Comment by Odrakir1** [comment]
ci:test:sf

**Comment by Odrakir1** [comment]
ci:test:relevant

**Comment by wanderlast** [comment]
Forwarded here: https://github.com/liferay-echo/liferay-portal/pull/5524

**Pull request closed without merging**